agents-manage-ui/src/components/agent/sidepane/sidepane.tsx

```diff
@@ -183,6 +185,8 @@ export function SidePane({
     credentialLookup,
     subAgentExternalAgentConfigLookup,
     subAgentTeamAgentConfigLookup,
+    // Rerender sidepane when errors changes
+    errors,
```

Comment:
now we can see errors in sidepane

https://github.com/user-attachments/assets/ce15c664-8479-4a0f-9b8b-947b892b8ea1